src/ppom-controller.ts

```diff
@@ -1031,6 +1031,10 @@ export class PPOMController extends BaseController<
       );
     }
 
+    if (this.#chainId !== ETHEREUM_CHAIN_ID) {
+      return undefined;
```

Comment:
```suggestion
      return;
```

Reply:
We need to specify the return value due to [this linting rule](https://eslint.org/docs/latest/rules/consistent-return). It might be worth having a discussion if we want to enable `treatUndefinedAsUnspecified` (though this is beyond the scope of this task).